### ISSUE TITLE
About the attention loss hyper-parameter，how to choose α-att?

### ISSUE THREAD

**Issue by MinZhangm** [miraiaroha/ACAN#2]
Hello,in your code the α-att is setted to 0, you haven't use the attention loss to train the network?
How to choose the value of the α-att？
When i set it equals 0.1,the total loss won't decrease when i train the net. So, could you please give me some tutorial?

**Comment by miraiaroha** [comment]
We found that adding the attention loss at the beginning of training is hard to converge as the extracted feature at the early stage of training is disordered, so the self-attention model cannot capture the meaningful context. You can (1) increasing α-att in the training process; (2) finetuning a trained model with attention loss.
